pkg/upgrade/upgrade.go

```diff
@@ -158,6 +167,10 @@ func (u *Upgrade) Run(ctx context.Context) error {
 		return errors.Join(err, errors.New("could not find install plan"))
 	}
 
+	if err := u.ensureEverestJWTIfNotExists(ctx); err != nil {
+		return err
+	}
```

Comment:
Should we do this conditionally? The user may use this CLI version to upgrade from 0.9 to 0.10 and in that case we don't want to run this code. We only want to run it if upgrading to 0.11.